requirements/pip.txt

```diff
@@ -1,49 +1,47 @@
 # Base packages
-pip==18.0
+pip==18.1
 appdirs==1.4.3
-virtualenv==16.0.0
+virtualenv==16.1.0
 docutils==0.14
-Sphinx==1.7.8
-sphinx_rtd_theme==0.4.1
-sphinx-tabs==1.1.7
+Sphinx==1.8.1
+sphinx_rtd_theme==0.4.2
+sphinx-tabs==1.1.8
 # Required to avoid Transifex error with reserved slug
 # https://github.com/sphinx-doc/sphinx-intl/pull/27
 git+https://github.com/agjohnson/sphinx-intl.git@7b5c66bdb30f872b3b1286e371f569c8dcb66de5#egg=sphinx-intl
 
 Pygments==2.2.0
 
-mkdocs==0.17.3
+mkdocs==1.0.4
 # mkdocs requires markdown
 # Markdown 3.0 breaks with older Django Rest Framework
 Markdown<3.0
 
 django==1.11.16
 six==1.11.0
-future==0.16.0
-
-# django-tastypie 0.14.0 drops support for django 1.9
-django-tastypie==0.13.3
-
+future==0.17.1
+django-tastypie==0.14.2
 django-guardian==1.4.9
-django-extensions==2.1.2
+django-extensions==2.1.3
 
-# djangorestframework 3.7.x drops support for django 1.9.x
-djangorestframework==3.6.4
+djangorestframework==3.9.0
 
 # Filtering for the REST API
-# django-filter>2.0 require Django 1.11
-# https://django-filter.readthedocs.io/en/master/guide/migration.html#migrating-to-2-0
-# Besides the guide, check this comment for the migration
-# https://github.com/rtfd/readthedocs.org/pull/4318#discussion_r214163531
-django-filter==1.1.0
+# When Python2 gets deprecated we can upgrade django-filter
+# >   from .filterset import FilterSet
+# E     File "/home/travis/build/rtfd/readthedocs.org/.tox/py27/lib/python2.7/site-packages/django_filters/filterset.py", line 184
+# E       def __init__(self, data=None, queryset=None, *, request=None, prefix=None):
+# E                                                     ^
+# E   SyntaxError: invalid syntax
+django-filter<2.0.0
```

Comment:
I guess the above comments

```python
filter_fields = ('slug',)  # django-filter<2.0.0
filterset_fields = ('slug',)
```
are because of this? I think is better to just create an issue, rather than _hiding_ it on the code

Reply:
We can't upgrade to 2.0.0 yet because we are still running tests with Py2. Once we remove that, we can upgrade without touching the code.

The comments added into the code already prepared it for 2.0.0.

Reply:
I prefer to prepare the code to work with both versions now and and we upgrade, we can remove the lines that are not needed.